services/src/test-library/src/main/java/org/openkilda/testing/service/database/Database.java

```diff
@@ -54,6 +57,8 @@
 
     Flow getFlow(String flowId);
 
+    Collection<TransitVlan> getTransitVlan(PathId forwardPathId, PathId reversePathId);
```

Comment:
since it returns a _collection_, the method name should be in plural: getTransitVlans

Reply:
fixed